wit/basic-producer.wit

```diff
@@ -0,0 +1,6 @@
+default interface basic-producer { 
+    use pkg.types.{message, error, sender}
+
+    send-message: func(s: sender, m: message) -> result<_, error>
```

Comment:
I would use `send-batch` for the method that takes a list. `send-many` says "multiple sends in a loop", but what you'll want the provider to do is to do a batched transfer.